scripts/test

```diff
@@ -15,7 +15,6 @@ git config user.name "Test"
 ./gootstrap -group testgroup -name testservice
 
 echo "generated files on: ${tmpdir}"
-tree
```

Comment:
Looks nice. Why remove it?

Reply:
We thought that not everybody would have this installed (but that was my cas :rofl: ) maybe we can put it back? @katcipis

Reply:
Ah yeah, it's a point as shell.. I said what I mean, looks nice but not a requirement. 

Maybe `command tree && tree` would do the trick? But you can ignore if you want

Reply:
yeah maybe, but one can always go to the dir the test created and run tree to investigate (or investigate some other way).